src/Migrate/Migrate.Autorest/custom/New-AzMigrateServerReplication.ps1

```diff
@@ -471,7 +471,9 @@ function New-AzMigrateServerReplication {
             $null = $PSBoundParameters.Remove('ResourceName')
             $null = $PSBoundParameters.Remove('SubscriptionId')
             $null = $PSBoundParameters.Add('Location', $TargetRegion)
-            $allAvailableSkus = Get-AzVMSize @PSBoundParameters -ErrorVariable notPresent -ErrorAction SilentlyContinue
+            #Get-AzVMSku -Location is deprecated, replicate using Get-AzComputeResourceSKU and a where clause
+            $null = $PSBoundParameters.Add('Location', $TargetRegion)
```

Comment:
duplicated line.

Reply:
Sorry on that, do you need me to redo the  PR?

Reply:
Please add a new commit to this PR.